core/src/main/java/hudson/lifecycle/UnixLifecycle.java

```diff
@@ -50,16 +52,12 @@
  * @since 1.304
  */
 public class UnixLifecycle extends Lifecycle {
+
+    @NonNull
     private List<String> args;
-    private Throwable failedToObtainArgs;
 
-    public UnixLifecycle() throws IOException {
-        try {
-            args = JavaVMArguments.current();
-        } catch (UnsupportedOperationException | LinkageError e) {
-            // can't restart / see JENKINS-3875
-            failedToObtainArgs = e;
-        }
```

Comment:
This was based on the old implementation, which used Akuma. That was ripped out a long time ago, and these exceptions shouldn't be thrown in the current implementation. But just to be on the safe side, we're now catching `Throwable` in the caller, so this shouldn't result in a regression in any case.